test/sales_engine_test.rb

```diff
@@ -63,13 +63,12 @@ def test_it_has_an_item_parser
   end
 
   def test_it_passes_the_correct_data_location_to_each_parser
-    item_data_location = "./data/fixtures/items.csv"
-    merchant_data_location = "./data/fixtures/merchants.csv"
-    dataset_locations = {merchant: merchant_data_location, item: item_data_location }
+    dataset_locations = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [145/80]<br>Space inside { missing.<br>Space inside } missing.